pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -654,6 +660,7 @@ self: super:
           "--with-xkb-bin-directory=${self.xkbcomp}/bin"
           "--with-xkb-path=${self.xkeyboardconfig}/share/X11/xkb"
           "--with-xkb-output=$out/share/X11/xkb/compiled"
+          "--with-log-dir=/var/log"
```

Comment:
Why do we override this at compile-time? It's probably not necessary to change anything here (except maybe add a clarification in a comment, depending on the reason), just asking out of curiosity.

Reply:
It defaults to $out/var and that seems to get hard-coded into the binary IIRC. Not a good location for trying to write logfiles, especially when a write failure causes a crash ;)

Reply:
Okay. Its always [preferable](https://mtlynch.io/code-review-love/#4-answer-questions-with-the-code-itself) to explain those things in a comment, but it's clear enough here and not important enough for another round of review. Thank you for the explanation :)